featurebyte/service/feature_store_warehouse.py

```diff
@@ -106,6 +131,37 @@ async def list_schemas(
         except db_session.no_schema_error as exc:
             raise DatabaseNotFoundError(f"Database {database_name} not found.") from exc
 
+    @staticmethod
+    def _is_visible_table(table_name: str, filter_featurebyte_tables: bool) -> bool:
+        if table_name.startswith("__"):
+            return False
+        if not filter_featurebyte_tables:
+            return True
+        # quick filter for materialized tables
+        if "TABLE" not in table_name:
+            return False
```

Comment:
Curious that 
```
        for prefix in MaterializedTableNamePrefix.visible():
            if table_name.startswith(prefix):
                return True
```
should be quite fast, right? Do we need this?

Reply:
The majority of tables are not materialized tables (most are tile tables). With this check we can reject these using a string comparison operation instead of multiple checks against each prefix